prod/docker-compose.prod.yml

```diff
@@ -39,18 +39,18 @@ services:
   django:
     build:
       context: ..
-      dockerfile: ./django.Dockerfile
+      dockerfile: ./prod/django.Dockerfile
```

Comment:
Can't we use a relative path here? This file is already in `/prod/`

Reply:
the dockerfile is relative to the context in docker compose.  The context is set to be the parent directory so the docker file needs to be under the build.  The env file is relative to the docker-compose.yml file location.  It's an annoying thing we typically avoid by having the docker compose files in the root directory of an application.

https://docs.docker.com/reference/compose-file/build/#illustrative-example